context-free.md

```diff
@@ -1,22 +1,26 @@
 # Context Free
 
 ##  Who is this programming language for?
+Context Free seems to target programmers who are also interested in creating art (computer-generated art is a pretty interesting, though unfortunately small, field).
 
 
 ## What is easy to do in this language? Why is it easy?
+Context Free specializes in drawing images through recursion. It is really easy to spawn more instances of the shape you're drawing through recursion, and mutual recursion allows for even more diversity in the kinds of images generated.
 
 
 ## What is hard to do in this language? Why is it hard?
+Since the output of this program is drawings, this language would not be good for any non-visual task and is limited in the visual tasks it can accomplish. Even drawing stuff other than repetitive / recursive drawings can be difficult, at least before the significant changes in the v3 release.
```

Comment:
Are there specific types of images that are hard. Its better to think of concrete examples than just think of things its good at and say everything else is hard.